### ISSUE TITLE
Selection sort in C++

### ISSUE THREAD

**Issue by venturo-raj7** [FOSS-UCSC/FOSSALGO#659]
#### This issue is about [Algo/DS Name](link to resource for the Algo/DS)

-   [ ] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [ ] Assign this issue to me (I want to work on this)

   -   [ ] I ensure that I am not already assigned to 2 or more issues
  
-   [x] I will send my PR only after I'm assigned to this issue by any maintainer

**Comment by RavinduSachintha** [comment]
@venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.

**Comment by venturo-raj7** [comment]
> @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.

Do I need to create pull request again please guide me through

**Comment by venturo-raj7** [comment]
> > @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.
> 
> Do I need to create pull request again please guide me through



> @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.

Also please tell can I raise issue again for other pull request can I clear them one by one?

**Comment by RavinduSachintha** [comment]
Due to you have closed the PR related to this issue, you can start from the beginning and send us a PR for this one. after we finish here everything, you can request for another issue and work on it.

**Comment by RavinduSachintha** [comment]
> > > @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.
> > 
> > 
> > Do I need to create pull request again please guide me through
> 
> > @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.
> 
> Also please tell can I raise issue again for other pull request can I clear them one by one?

I have closed other PRs. but not issues. It is not required to close issues because they are not duplicated. but we will not be able to assign you for any of that before completing this one.

**Comment by venturo-raj7** [comment]
> > > > @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.
> > > 
> > > 
> > > Do I need to create pull request again please guide me through
> > 
> > 
> > > @venturo-raj7 as our contribution guidelines, anyone will be allowed only to be assigned to a single issue per time. And also, creating unnecessary PRs without properly assigned to an issue is kind of a violation of those guidelines. Therefore, I will assign you to this issue and other issues that you have raised will remain as open. I have to close unnecessary PRs without considering for reviewing. Hope you understand and good luck! Thank you.
> > 
> > 
> > Also please tell can I raise issue again for other pull request can I clear them one by one?
> 
> I have closed other PRs. but not issues. It is not required to close issues because they are not duplicated. but we will not be able to assign you for any of that before completing this one.

Thanku so much sir after this pr I will first get other issue assigned then submit another pr
